### ISSUE TITLE
The last character is out of the rectangle

### ISSUE THREAD

**Issue by slevy85** [geongeorge/Canvas-Txt#20]
Hi, 
I noticed that sometimes the last character of the word is out of the rectangle :  
![image](https://user-images.githubusercontent.com/18260229/86516360-c82ae100-be17-11ea-8671-c632c7cbda58.png)

This happens when the last character which is out the rectangle is also the last character of a word:  
```
while (textpixlen < linelen) {
            textlen++
            texttoprint = temptext.substr(0, textlen)
            textpixlen = ctx.measureText(temptext.substr(0, textlen)).width
          }
```
This can be fixed by removing the last character after this loop :
```
textlen--
texttoprint = texttoprint.substr(0, textlen)
```

Can I make a PR for this ?

It is important for me because when the text is justified we see some lines with a character which is not aligned.

**Comment by geongeorge** [comment]
"This can be fixed by removing the last character after this loop"

What is the end result? the word ends up in a new line?

**Comment by slevy85** [comment]
Yes exactly

**Comment by geongeorge** [comment]
Looks perfect

**Comment by geongeorge** [comment]
Solved by merging  #19
